app/cmd/config_remove.go

```diff
@@ -45,9 +45,10 @@ func removeJenkins(name string) (err error) {
 	if index == -1 {
 		err = fmt.Errorf("Cannot found by name %s", name)
 	} else {
-		config.JenkinsServers[index] = config.JenkinsServers[len(config.JenkinsServers)-1]
-		config.JenkinsServers[len(config.JenkinsServers)-1] = JenkinsServer{}
-		config.JenkinsServers = config.JenkinsServers[:len(config.JenkinsServers)-1]
+		// config.JenkinsServers[index] = config.JenkinsServers[len(config.JenkinsServers)-1]
+		// config.JenkinsServers[len(config.JenkinsServers)-1] = JenkinsServer{}
+		// config.JenkinsServers = config.JenkinsServers[:len(config.JenkinsServers)-1]
+		config.JenkinsServers = append(config.JenkinsServers[:index], config.JenkinsServers[index+1:]...)
```

Comment:
There're more works need to do if the user removes the current Jenkins config item. For example, we can let the user change current Jenkins first.

Reply:
Yes, I did take that piece into consideration and revised it before. Later, after recovering the file, I forgot to modify it.

Reply:
Please remove the comment lines.